### ISSUE TITLE
Request: obs-websocket command?

### ISSUE THREAD

**Issue by plamenbe** [Niek/obs-web#97]
Is it possible somehow to run arbitrary command, supported by obs-websocket protocol?
I mean something like a simple console box and similarly to "/command" option from ["Command line tool for obs-websocket"](https://obsproject.com/forum/resources/command-line-tool-for-obs-websocket-plugin-windows.615/)

Thank you in advance!

**Comment by Niek** [comment]
It would not be very hard to allow sending arbitrary commends through the `sendCommand()` function, but I don't really see how to implement that without adding yet another button that 99% of the people won't use. Maybe we can expose it to the devtools so you can run it from there?

**Comment by plamenbe** [comment]
Yes, I think, that this solution with devtools would be good enough.
Thank you!

**Comment by Niek** [comment]
Added in https://github.com/Niek/obs-web/commit/ed86d31ab4a211f7271c1042c7708af528bc73f9 - you can now send commands from devtools:
<img width="704" alt="Screenshot 2022-08-30 at 16 42 58" src="https://user-images.githubusercontent.com/213140/187467207-196884b4-eb6f-4a20-8b43-a8077d165d4e.png">

**Comment by plamenbe** [comment]
It is great and very useful !!!
